### ISSUE TITLE
Redundant Boarding Rate Setting

### ISSUE THREAD

**Issue by MicahBCode** [flybywiresim/a32nx#7687]
### Aircraft Version

Development

### Description

{
      "built": "2023-01-02T11:51:15+00:00",
      "ref": "refs/heads/master",
      "sha": "2b2721a36a3e021db354f487a8222f00d4caaca5",
      "actor": "frankkopp",
      "event_name": "manual",
      "pretty_release_name": "master:2b2721a3"
}

### References (optional)

The boarding rate can be set in the settings page as well as the payload tab on the ground page in the EFB.
To keep it consistent to other rate settings, like the refueling rate, I would recommend removing the boarding rate setting from the settings page.

This keeps the EFB settings consistent and removes redundancy as well as complexity.

### Discord Username (optional)

Micah#3002

**Comment by patmack14** [comment]
Is this request still on the wishlist? A little confused based on discussion in #7688

**Comment by MicahBCode** [comment]
Well, I think its unneccessary redundancy to have the boarding rate setting twice and I think it's not necessary to have the boarding rate setting in that place as well since you always start the boarding via the payload page, so you will be at the correct page anyways if you wish to change the setting prior to boarding.

Let me know what you guys think on that.

**Comment by tracernz** [comment]
I agree with you, but I'd suggest even further doing away with the rate settings now that we have an instant load button that finishes everything whenever you're ready.

**Comment by MicahBCode** [comment]
You mean that it uses "realtime" setting per default and if the user wants to finish early he has the "instant load" button? I would support that solution as well. Seems legit to me.

**Comment by ImPlasmaTbh** [comment]
assign me onto this issue please.
